src/main/java/games/strategy/engine/framework/EngineVersionProperties.java

```diff
@@ -37,9 +38,9 @@ private EngineVersionProperties() {
   private EngineVersionProperties(final Properties props) {
     latestVersionOut =
         new Version(props.getProperty("LATEST", ClientContext.engineVersion().toStringFull(".")));
-    link = props.getProperty("LINK", "http://triplea-game.github.io/");
-    linkAlt = props.getProperty("LINK_ALT", "http://triplea-game.github.io/download/");
-    changelogLink = props.getProperty("CHANGELOG", "http://triplea-game.github.io/release_notes/");
+    link = props.getProperty("LINK", UrlConstants.TRIPLEA_WEBSITE.toString());
+    linkAlt = props.getProperty("LINK_ALT", UrlConstants.TRIPLEA_WEBSITE + "download/");
```

Comment:
Any objection to converting the download and release note links to be constants as well? The constants class is nice to aggregate all URL constants and to avoid any client code from constructing or knowing about URLs. With the string concatenation I think we violate a bit, we are spreading the knowledge of the URL between both classes rather than just one.